src/app/_components/most-popular-section/tabs.tsx

```
@@ -0,0 +1,39 @@
+              <CardDescription>
+                happy birthday to yooooooooou
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="grid gap-6">
+             hhhhhhhhhhhh
+            </CardContent>
+          </Card>
+        </TabsContent>
+      </Tabs>
+  )
+}
```

ESLint

[error] Missing semicolon.

(semi)